sagemaker-python-sdk/mxnet_horovod_maskrcnn/source/requirements.txt

```diff
@@ -0,0 +1,5 @@
+gluoncv==0.8.0b20200723 
```

Comment:
GluonCV 0.8.0 has been released. so we can use the stable version instead of nightly.

Reply:
Updated